morningstar/good_morning.py

```diff
@@ -20,31 +20,32 @@
 """Module for downloading financial data from financials.morningstar.com.
 """
 
-from __future__ import with_statement
 from __future__ import absolute_import
```

Comment:
I'll delete this; but I was going to make this code work with Python 2 eventually (next plan) for backwards compatibility.  

http://python-future.org/quickstart.html